test-ets.sh

```diff
@@ -1,32 +1,52 @@
-# this script is meant to run the Ethereum Test Suite (https://github.com/ethereum/tests) on CircleCI
-# to skip these tests, set SKIP_ETS_TEST variable CircleCI UI
-# to differentiate by branch, use CIRCLE_BRANCH variable in the conditional below
-
-echo "current branch: $BUILDKITE_BRANCH"
-
-echo "memory:"
-free -m
+#!/usr/bin/env bash
 
 if [ -z "$IN_NIX_SHELL" ]; then
     export SBT_NIX="-Dnix=true"
 fi
 
-# at least 1.5G of heap space and 10M of stack size is required for ETS.
-export JAVA_OPTS="-Xmx2g -Xss16m -XX:MaxMetaspaceSize=512m"
-export SBT = "sbt -v -mem 2048 $SBT_NIX";
-
-if [ -z $SKIP_ETS_TESTS ]; then
-    git submodule init;
-    git submodule update;
-    if [ "$BUILDKITE_BRANCH" == "master" -o -n "$RUN_FULL_ETS" ]; then
-        export JAVA_OPTS="-Xmx3g -Xss16m -XX:MaxMetaspaceSize=512m"
-        echo "running full ETS"
-        $SBT "ets:testOnly * -- -DuseLocalVM=true -Dexg=vmPerf*";
-    else
-        echo "running a subset of ETS"
-        $SBT "ets:testOnly *VMSuite -- -Dexg=vmPerf*" &&
-        $SBT "ets:testOnly *BlockchainSuite -- -DuseLocalVM=true -Ding=bcForkStress*,bcMulti*,bcState*,bcTotalDiff*,bcValidBlock*,Transition*";
-    fi
-else
-    echo "SKIP_ETS_TESTS variable is set - skipping the tests";
-fi
+git submodule init
+git submodule update
```

Comment:
I think we need to automate this for the repository

Reply:
How?

Reply:
good question. I think that easiest would be a task that runs every so often and checks if the modules are out of date and creates a PR is they are. I don't think this can be done now, just making a mental note.

btw I meant keeping the submodules in sync (`git submodule update --remote --merge`) and I see why this comment caused confusion. apologies

Reply:
Ah, yeah, these commands only pull in the submodules in the checked out repo.

Updating those submodules might introduce breaking changes. Something like Scala Steward for this would be ideal...